### ISSUE TITLE
audio driven

### ISSUE THREAD

**Issue by agupta54** [zjumsj/GaussianBlendshapes#12]
Is it possible to make the face rendering audio driven? I was trying to use audio conditioned expression and jaw parameters from an audio to expression model instead of tracked ones during testing. The rendered flame mesh had correct lip movements, but the gaussian avatar had a lot of visual artefacts. I suspect this is because it doesn't generalise to blend weights outside the training distribution. Could you suggest a solution to this?

**Comment by zjumsj** [comment]
This is a known issue that also occurs in cross-identity reenactment, and it may be caused by several factors. 

One reason is that some identity information is incorrectly encoded into the expression coefficients. A natural-looking neutral face does not correspond to all-zero expression coefficients, which results in poorer transfer performance. The solution is to compute the average expression coefficients from the training set to represent the 'real' neutral expression and then normalize the expression coefficients accordingly.

Additionally, if the test set extends too far beyond the training set's parameter space, generalization issues may arise. To mitigate this, you can increase the diversity in the training set or establish a PCA space from the training set's expression parameters. This involves projecting the test expressions onto the PCA space and then back-projecting them. This trick can reduce artifacts, although it may also eliminate some details in the expressions.

**Comment by shawnFuu** [comment]
> This is a known issue that also occurs in cross-identity reenactment, and it may be caused by several factors.
> 
> One reason is that some identity information is incorrectly encoded into the expression coefficients. A natural-looking neutral face does not correspond to all-zero expression coefficients, which results in poorer transfer performance. The solution is to compute the average expression coefficients from the training set to represent the 'real' neutral expression and then normalize the expression coefficients accordingly.
> 
> Additionally, if the test set extends too far beyond the training set's parameter space, generalization issues may arise. To mitigate this, you can increase the diversity in the training set or establish a PCA space from the training set's expression parameters. This involves projecting the test expressions onto the PCA space and then back-projecting them. This trick can reduce artifacts, although it may also eliminate some details in the expressions.

So nice of you to give us detailed analysis and solutions! I wonder whether we can drive the avatar for unseen expressions and poses. Does Large dataset with different identities help? Can you give us any advice?
